index.html

```diff
@@ -46,6 +48,15 @@
             <button class="function-button">+</button>
           </div>
         </div>
+
+      <div class="spacing"></div>
```

Comment:
```suggestion
      <hr>
```
Seems to be better cause it is only a styling element with no semantic meaning. Also check the changes in `styles-drag-drop.css`